### PR TITLE
Redesign evaluations landing page

### PR DESCRIPTION
**Pull request by samnoyes**

### CODE REVIEW

File: docs/evaluation/index.mdx [hunk 1]
```diff
@@ -2,26 +2,101 @@
...
-Having a proper evaluation framework that you trust will allow you to:
+## What are evaluations?
 
-- Experiment with different prompts and accurately determine if changes improve performance
-- Iterate on aspects your RAG architecture without guesswork
-- Quickly decide if it is worth upgrading to a new, recently release model
-- Determine if you can use a cheaper, faster model in place of a more expensive one and get similar performance
-- Flag with a model behind an API changed suddenly in an unexpected way
-- Have confidence your application will not produce biased or unwanted responses
+Evaluations allow you to understand the performance of your LLM application over time. At its core, an evaluation is a function that takes in a set of inputs and outputs from your chain, agent, or model,
```
Comment (hinthornw) [inline docs/evaluation/index.mdx:11]: Can return multiple scores

File: docs/evaluation/index.mdx [hunk 2]
```diff
@@ -2,26 +2,101 @@
...
 
-- Experiment with different prompts and accurately determine if changes improve performance
-- Iterate on aspects your RAG architecture without guesswork
-- Quickly decide if it is worth upgrading to a new, recently release model
-- Determine if you can use a cheaper, faster model in place of a more expensive one and get similar performance
-- Flag with a model behind an API changed suddenly in an unexpected way
-- Have confidence your application will not produce biased or unwanted responses
+Evaluations allow you to understand the performance of your LLM application over time. At its core, an evaluation is a function that takes in a set of inputs and outputs from your chain, agent, or model, 
+and returns a score. This score may be based on comparing the outputs with reference outputs (e.g. with string matching or using an LLM as a judge).
+However, there are also evaluators that don't require a reference output - for example, one that checks if the output is blank, which is a common failure mode in LLM applications.
```
Comment (hinthornw) [inline docs/evaluation/index.mdx:13]: "for example, one that checks if the output is blank, which is a common failure mode in LLM applications." maybe not all that common: maybe a more relevant one is like "output is valid json" or "conforms to specific schema"

File: docs/evaluation/index.mdx [hunk 3]
```diff
@@ -2,26 +2,101 @@
...
+LangSmith makes it easy to seamlessly run evaluations against your application via `Datasets`, which are made up of `Examples`.
```
Comment (hinthornw) [inline docs/evaluation/index.mdx:14]: "easy" and "seamlessly" redundant. We should eschew adjectives where possible: show don't tell


"LangSmith helps you evaluation your application against `Datasets`, which are made up of `Examples`." maybe?

Comment (samnoyes) [reply docs/evaluation/index.mdx]: good point, changed to "LangSmith allows you to run evaluations on your application via Datasets, which are made up of Examples." That good?

**Review by hinthornw — APPROVED** [review]

**Comment by hinthornw** [comment]
Step in the right direction

**Pull request merged**